ign_rviz_plugins/res/qml/GlobalOptions.qml

```diff
@@ -104,6 +104,23 @@ Item {
         }
       }
     }
+
+    RowLayout {
+      width: parent.width
+      spacing: 10
+
+      Label {
+        id: "status"
+        text: GlobalOptions.tfStatus.status
+        color: GlobalOptions.tfStatus.color
+      }
+
+      Label {
+        id: "message"
+        text: GlobalOptions.tfStatus.message
+        color: GlobalOptions.tfStatus.color
+      }
+    }
```

Comment:
Can you look into the text wrapping issue?
The FixedFrame combo-box drop-down menu is not selectable when the plugin width < text width.

Reply:
I have added `elide` property to the second `Label` element:
Default | Expanded
--- | ---
![Screenshot from 2021-01-26 20-52-06](https://user-images.githubusercontent.com/39728574/105865245-bbbadf80-6018-11eb-870a-ce5e38a6dae2.png) | ![Screenshot from 2021-01-26 20-52-22](https://user-images.githubusercontent.com/39728574/105865260-bf4e6680-6018-11eb-983a-f7b66560db13.png)